netflix-clone-yt-master/src/app/pages/login/login.component.ts

```diff
@@ -0,0 +1,43 @@
+declare var google: any;
+import { Component, OnInit, inject } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-login',
+  standalone: true,
+  imports: [CommonModule],
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.scss']
+})
+export class LoginComponent implements OnInit {
+  private router = inject(Router);
+  ngOnInit(): void {
+    google.accounts.id.initialize({
```

Comment:
Consider moving hardcoded client_id to environment variables for better security and configurability.